unitOneAssessmentPractice.js

```diff
@@ -14,22 +16,43 @@ let assert = require('assert')
 // Uncomment out the next line to test your solution
 // runQ2Tests()
 
+const containsSeven = (num) => {
+  let stringToNum = num.toString()
```

Comment:
technically it's numToString :) Be careful when naming your variables and functions - "incorrect" names might confuse you later.